app/experimenter/nimbus-ui/src/components/PageHome/Directory.tsx

```diff
@@ -0,0 +1,84 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { useEffect, useState } from "react";
+import { useQuery } from "@apollo/client";
+import { GET_ALL_EXPERIMENTS } from "../../gql/experiments";
+import { GetAllExperiments_experiments } from "../../types/GetAllExperiments";
+import { humanDate, addDaysToDate } from "../../lib/dateUtils";
+
+const Directory: React.FunctionComponent = () => {
+  const { data } = useQuery<{
+    experiments: GetAllExperiments_experiments[];
+  }>(GET_ALL_EXPERIMENTS);
+
+  if (!data) {
+    return <div />;
+  }
+  const { experiments } = data;
+  return (
+    <div>
+      <table className="table table-sm">
+        <thead>
+          <tr>
+            <th>Experiment</th>
+            <th>Type</th>
+            <th>Status</th>
+            <th>Start Date</th>
+            <th>End Date</th>
+            <th>Enrollment End Date</th>
+            <th>Results</th>
+          </tr>
+        </thead>
+        <tbody>
+          {experiments.map((experiment) => {
+            const {
+              name,
+              slug,
+              status,
+              startDate,
+              proposedDuration,
+              proposedEnrollment,
+              endDate,
+              featureConfig,
+            } = experiment;
+            console.log(proposedEnrollment);
+            return (
+              <tr key={slug}>
+                <td>
+                  <a href={`/experiments/nimbus/${slug}`}>{name}</a>
+                </td>
+                <td>
+                  {featureConfig ? featureConfig.slug : "Simple Experiment"}
+                </td>
+                <td>{status}</td>
+                <td>{startDate ? humanDate(startDate) : "-"}</td>
+                <td>
+                  {endDate
+                    ? humanDate(endDate)
+                    : startDate
+                    ? humanDate(addDaysToDate(startDate, proposedDuration))
+                    : `+${proposedDuration} days`}
```

Comment:
Looking at the [GraphiQL dashboard](https://stage.experimenter.nonprod.dataops.mozgcp.net/api/v5/graphql#query=%7B%0A%20%20experiments(limit%3A10%2C%20status%3ADraft)%20%7B%0A%20%20%20%20id%0A%20%20%20%20owner%20%7B%0A%20%20%20%20%20%20id%0A%20%20%20%20%7D%0A%20%20%20%20status%0A%20%20%20%20name%0A%20%20%20%20slug%0A%20%20%20%20publicDescription%0A%20%20%20%20isPaused%0A%20%20%20%20%0A%20%20%20%20proposedDuration%0A%20%20%20%20proposedEnrollment%0A%20%20%20%20populationPercent%0A%20%20%20%20totalEnrolledClients%0A%20%20%20%20firefoxMinVersion%0A%20%20%20%20%0A%20%20%20%20application%0A%20%20%20%20projects%20%7B%0A%20%20%20%20%20%20id%0A%20%20%20%20%7D%0A%20%20%20%20hypothesis%0A%20%20%20%20featureConfig%20%7B%0A%20%20%20%20%20%20id%0A%20%20%20%20%7D%0A%20%20%20%20referenceBranch%20%7B%0A%20%20%20%20%20%20featureValue%0A%20%20%20%20%7D%0A%20%20%20%20bucketRange%20%7B%0A%20%20%20%20%20%20start%0A%20%20%20%20%20%20count%0A%20%20%20%20%20%20isolationGroup%20%7B%0A%20%20%20%20%20%20%20%20id%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%20%20treatmentBranches%20%7B%0A%20%20%20%20%20%20featureValue%0A%20%20%20%20%7D%0A%20%20%20%20primaryProbeSets%20%7B%0A%20%20%20%20%20%20id%0A%20%20%20%20%7D%0A%20%20%20%20secondaryProbeSets%20%7B%0A%20%20%20%20%20%20id%0A%20%20%20%20%7D%0A%20%20%20%20%0A%20%20%20%20readyForReview%20%7B%0A%20%20%20%20%20%20message%0A%20%20%20%20%20%20ready%0A%20%20%20%20%7D%0A%20%20%20%20monitoringDashboardUrl%0A%20%20%20%20startDate%0A%20%20%20%20endDate%0A%20%20%20%20%0A%20%20%7D%0A%7D%0A), it looks like experiments are in a default state where `proposedDuration` and `proposedEnrollment` are `null` (even though they appear as 7 or 21 in the form initially). Not sure if we'd have to account for that here, or if it'd end up rendering `+null days` 

```js
        "proposedDuration": null,
        "proposedEnrollment": null,
```

My half-assed testing in a browser console didn't see any errors in `addDaysToDate()` where the `days` argument would potentially be `null` in this case and we end up doing something like `date.setDate(date.getDate() + null)`.